jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/EventsHandler.java

```diff
@@ -309,11 +309,30 @@ protected void onResponseTrailersComplete(Request request, HttpFields trailers)
      *
      * @param request the request object. The {@code read()}, {@code demand(Runnable)} and {@code fail(Throwable)} methods must not be called by the listener.
      * @param failure if there was a failure to complete
+     * @deprecated Override {@link #onComplete(Request, int, HttpFields, Throwable)} instead.
      */
+    @Deprecated
```

Comment:
Now that this method is deprecated, please update the documentation as well, namely `HTTPServerDocs.eventsHandler()`.
Also look at the `*.adoc` text referring to that example, to see if it can be expanded to talk about the new parameters.

Reply:
Done. There wasn't much to change in the documentation, everything seems to still be accurate in the adoc files, only `HTTPServerDocs.eventsHandler()` needed some modifications.